packages/frint/src/App.ts

```diff
@@ -0,0 +1,418 @@
+import { Container, createContainer, Provider, resolveContainer } from 'frint-di';
+import find = require('lodash/find');
+import findIndex = require('lodash/findIndex');
+import get = require('lodash/get');
+import omit = require('lodash/omit');
+import 'rxjs/add/observable/of';
```

Comment:
`rxjs/add/*` has side-effects, and mutates the package globally.

what's happening here:

```js
// rxjs/add/observable/of.js

// pure side-effects free modules
const Observable = require('rxjs/Observable');
const of = require('rxjs/observable/of');

// mutation
Observable.of = of;
```

that's why we added an ESLint rule here to avoid these imports: https://github.com/frintjs/frint/blob/master/.eslintrc#L12

may be we can do something similar with TSLint?

Reply:
Yep, there is a `no-import-side-effect` rule, I added it to `tslint.json`.